wiki/Placeholders.md

```diff
@@ -4404,6 +4405,13 @@ Replace `XXX` with the enchantment.
 ```
 ----
 
+- ### **[TokensPlus](https://www.spigotmc.org/resources/tokensplus.90507/)**
+> NO DOWNLOAD COMMAND
```

Comment:
Add an empty line here to not have the code block as quote.

Reply:
Added empty line below >NO DOWNLOAD COMMAND